lib/passwordless/exceptions.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+module Passwordless
+  # Abstract passwordless errors from StandardError.
+  class PasswordlessError < StandardError; end
+  # Exception raised when a session is expired.
+  class ExpiredSessionError < PasswordlessError; end
```

Comment:
While this is not a bad idea, I think I'd prefer to just have the error be a subclass  to the controller. Like:

```ruby
module Passwordless
  class SessionsController < ApplicationController
    class ExpiredSessionError < StandardError; end
    # ...
```

As this is our only error so far and the controller is the only place it'll be raised, I don't think we need to centralize the errors just yet :)